src/proto/FunctionRpc.proto

```diff
@@ -240,6 +240,13 @@ message FunctionEnvironmentReloadRequest {
 message FunctionEnvironmentReloadResponse {
   // Status of the response
   StatusResult result = 3;
+
+  // After specialization, worker sends capabilities & metadata.
+  // A map of worker supported features/capabilities
+  map<string, string> capabilities = 2;
```

Comment:
Should these fields be `optional` so they aren't serialized for non-`dotnet-isolated` scenarios?

Reply:
[proto3](https://developers.google.com/protocol-buffers/docs/proto3) supports optional fields. I do not see we use it anywhere though. I vaguely remember asking this question to @liliankasem a while ago and obviously forgot what she told me. @liliankasem can you help :) ?

Reply:
The `optional` keyword will break things and we've seen this happen [before](https://github.com/Azure/azure-functions-language-worker-protobuf/pull/70#discussion_r897127958) - we shouldn't use it. IIRC everything in the proto is by default optional